sql/core/src/main/scala/org/apache/spark/sql/internal/SharedState.scala

```diff
@@ -20,6 +20,7 @@ package org.apache.spark.sql.internal
 import java.net.URL
 import java.util.Locale
 
+import scala.collection.Map
```

Comment:
please don't import it, just use full class name in the code below, the narrow down the scope.